Source/WebCore/rendering/svg/SVGTextLayoutEngine.cpp

```diff
@@ -277,13 +279,14 @@ void SVGTextLayoutEngine::finalizeTransformMatrices(Vector<SVGInlineTextBox*>& b
     AffineTransform textBoxTransformation;
     for (unsigned boxPosition = 0; boxPosition < boxCount; ++boxPosition) {
```

Comment:
You can make this also a modern C++ for loop:

```
void SVGTextLayoutEngine::finalizeTransformMatrices(Vector<SVGInlineTextBox*>& textBoxes)
{
    if (textBoxes.isEmpty())
        return;

    for (auto textBox : textBoxes) {
        auto textBoxTransformation = m_chunkLayoutBuilder.transformationForTextBox(textBox);
        if (textBoxTransformation.isIdentity())
            continue;

        for (auto& fragment : textBox->textFragments()) {
            ASSERT(fragment.lengthAdjustTransform.isIdentity());
            fragment.lengthAdjustTransform = textBoxTransformation;
        }
    }

    textBoxes.clear();
}
```

Reply:
Done.